tools/release/release.py

```diff
@@ -103,4 +103,8 @@ def build_argparse() -> argparse.ArgumentParser:
             "Environment variable `GH_ACCESS_TOKEN` and `GH_REPO_MILESTONE` must provider"
         )
 
-    print(args.func(ENV_ACCESS_TOKEN, ENV_MILESTONE))
+    try:
+        print(args.func(ENV_ACCESS_TOKEN, ENV_MILESTONE))
+    except Exception as e:
+        print(f"Please run 'python release.py -h' to get help, reason: {e}")
+        exit(1)
```

Comment:
We could directly check if `func` is in object `args` and reuse `print_help` method from ArgumentParser object to keep our code easier

```py
    if not hasattr(args, 'func'):
        arg_parser.print_help()
        exit(1)
    print(args.func(ENV_ACCESS_TOKEN, ENV_MILESTONE))
```

Reply:
Fixed. PTAL